edi_oca/models/edi_backend.py

```diff
@@ -579,6 +580,21 @@ def _exchange_receive(self, exchange_record):
             return component.receive()
         raise NotImplementedError()
 
+    def _process_received_content(self, exchange_record, content):
```

Comment:
IMO this should be done in `validate_data` and you can set the content of the file directly.
I would:
* keep the code as it was before but remove the check on the content value.
* set the value as it is on the record
* call validate data and raise the error there if necessary